M2/Macaulay2/d/gmp_aux.c

```diff
@@ -21,6 +21,15 @@ int mpfr_hash(mpfr_srcptr x) {
   return 777 + h * 3737 + x->_mpfr_exp + 11 * x->_mpfr_sign;
 }
 
+int mpfi_hash(mpfr_srcptr x) { // Added for MPFI
```

Comment:
This looks wrong: "mpfi_hash" should take an mpfi, not an mpfr.

Reply:
Why not define mpfi_hash the sum of the results of mpfr_hash on the two endpoints?